src/parser/core/healingEfficiency/HealingEfficiencyTracker.ts

```diff
@@ -84,7 +84,7 @@ class HealingEfficiencyTracker extends Analyzer {
 
     if (healingSpellIds) {
       for (const healingSpellId of healingSpellIds) {
```

Comment:
Someone changed this a while ago and it has probably been broken since. `of` != `in`